test/functional/feature_rbf.py

```diff
@@ -84,10 +85,8 @@ def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def run_test(self):
-        make_utxo(self.nodes[0], 1 * COIN)
-
-        # Ensure nodes are synced
-        self.sync_all()
+        self.wallet = MiniWallet(self.nodes[0])
+        self.wallet.scan_blocks(start=76, num=3)
```

Comment:
Previously,
We were scanning block 76 for `test_no_inherited_signaling(self)` and block 77 for `test_replacement_relay_fee(self)`

However in the PR,
We are scanning blocks 76,77, and 78.

Block 78 is not required to be scanned here, hence, we can rather have this as `self.wallet.scan_blocks(start=76, num=2)`

Reply:
Thanks, done.